### PR TITLE
Add contract verification guide to Developing Contracts

### PR DESCRIPTION
**Pull request by ElPaisano**
Addresses https://github.com/filecoin-project/filecoin-docs/pull/1952#issuecomment-1551015334 by adding a basic guide on contract verification to Developing Contracts. Based off of https://gist.github.com/trruckerfling/54918b8331f6fa10ae27cedd6afd9680

### CODE REVIEW

File: content/en/smart-contracts/developing-contracts/verify-contracts/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+### Procedure
+
+First
+
+1. Open Remix.
+
+1. In the **File Explorer** sidebar tab, under **contracts**, right click on your contract.
+
+1. From the menu, select **generate UM** to flatten `<contract-name>.sol` file and ensure all components and tasks are included.
```
[the remark this answers is not in the document]
Comment (trruckerfling) [reply content/en/smart-contracts/developing-contracts/verify-contracts/index.md:41]: adding screenshot:
Screen Shot 2023-04-21 at 4 56 01 PM

File: content/en/smart-contracts/developing-contracts/verify-contracts/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+1. In the menu, click **Download** to save the file. 
+
+1. Copy the address of your deployed contract. 
+
+1. Navigate to [Filfox](https://filfox.info/en/contract) as an example.
+
+1. In the form, enter the following information:
+   - The address of your deployed contract.
+   - The compiler version used in 
+   - The license type used in
```
Comment (trruckerfling) [inline content/en/smart-contracts/developing-contracts/verify-contracts/index.md:60]: Adding screenshot:

Screen Shot 2023-04-21 at 5 05 56 PM

File: content/en/smart-contracts/developing-contracts/verify-contracts/index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+1. Click **Continue** to upload your flattened .sol file. 
+
+1. Click **Verify and Publish**.
+
```
Comment (trruckerfling) [inline content/en/smart-contracts/developing-contracts/verify-contracts/index.md:65]: Adding screenshot:

Screen Shot 2023-04-21 at 5 07 39 PM

File: content/en/smart-contracts/developing-contracts/verify-contracts/index.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+   Success! Your contract is now verified.
+
```
Comment (trruckerfling) [inline content/en/smart-contracts/developing-contracts/verify-contracts/index.md:67]: Adding screenshot: 

Screen Shot 2023-04-21 at 5 08 47 PM

File: content/en/smart-contracts/developing-contracts/verify-contracts/index.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+1. In Filfox's search bar, enter your contract address.
+
```
Comment (trruckerfling) [inline content/en/smart-contracts/developing-contracts/verify-contracts/index.md:69]: Adding screenshot:

Screen Shot 2023-05-24 at 3 17 04 PM

File: content/en/smart-contracts/developing-contracts/verify-contracts/index.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+1. Select the **Contract** tab to you can view your contract's source code and ABI.
+
```
Comment (trruckerfling) [inline content/en/smart-contracts/developing-contracts/verify-contracts/index.md:71]: Screen Shot 2023-05-24 at 3 21 02 PM

File: content/en/smart-contracts/developing-contracts/verify-contracts/index.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+> You may also view other verified contracts' source code and ABI by heading back to Filfox contract verification page and clicking on 'Verified Contract List'. You can click into contract addresses and follow the steps above, in this section.
```
Comment (trruckerfling) [inline content/en/smart-contracts/developing-contracts/verify-contracts/index.md:72]: Screen Shot 2023-05-24 at 3 23 46 PM
Screen Shot 2023-05-24 at 3 25 20 PM

**Comment by ElPaisano** [comment]
> Added screenshots to include and light edits for accuracy. Please help review and merge, thank you!

@trruckerfling sorry for the confusion here, this PR was half-baked when you reviewed.

Finished up the PR. Rewrote overall flow and addressed some of your requested changes:

- "UM" -> "UML"
- Fixed the frontmatter `lead`
- Renamed sections

I tried to write the instructions in such a way that screenshots are not needed. Generally, I don't like screenshots in the docs because it generally means more maintenance since screenshots and UIs get out of date fast. So, I'd prefer not to include screenshots unless we feel it's absolutely needed.

**Comment by ElPaisano** [comment]
@johnnymatthews a few things here for you:

- what do you think RE https://github.com/filecoin-project/filecoin-docs/pull/1993#issuecomment-1569051392?
- where should this page live in the docs hierarchy? It's currently in Smart contracts -> developing contracts

Also, note that for merge purposes, this PR spun out of [https://github.com/filecoin-project/filecoin-docs/pull/1952#issuecomment-1551015334](https://github.com/filecoin-project/filecoin-docs/pull/1952#issuecomment-1551015334)

File: content/en/smart-contracts/developing-contracts/verify-contracts/index.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+- Access to one of the following contract verification tools.
+    - [Filfox](https://filfox.info/en/contract)
+    - [Starboard](https://fvm.starboard.ventures/explorer/verifier) 
+    - [Beryx](https://beryx.zondax.ch/contract_verifier)
+
+### Procedure
+
+First
+
+1. Open Remix.
```
[the remark this answers is not in the document]
Comment (ElPaisano) [reply content/en/smart-contracts/developing-contracts/verify-contracts/index.md:38]: Hey @trruckerfling this is a Markdown convention that the docs team (me and @johnnymatthews lol) try and keep across the docs. It renders as 1, 2, 3, ... in the UI

File: content/en/smart-contracts/developing-contracts/verify-contracts/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ElPaisano) [reply content/en/smart-contracts/developing-contracts/verify-contracts/index.md]: > edit to "generate UML"

Done ✅

File: content/en/smart-contracts/developing-contracts/verify-contracts/index.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+    parent: "smart-contracts-developing-contracts"
+    identifier: "verify-contracts-cff1cc77a90c8bea84857e1139981325"
+weight: 460
+toc: true
+aliases:
+    - "/developers/infrastructure/libraries/"
+---
+
+## Verify your contract
+If you are deploying your FEVM contract to mainnet, we highly recommend verifying your contract, to upkeep security and transparency on the network.
```
[the remark this answers is not in the document]
Comment (ElPaisano) [reply content/en/smart-contracts/developing-contracts/verify-contracts/index.md]: Addressed

**Review by trruckerfling — APPROVED** [review]
good to go!

**Review by johnnymatthews — APPROVED** [review]
Looks good!

**Pull request merged**